src/python_testing/TC_ACE_1_5.py

```diff
@@ -0,0 +1,164 @@
+#
+#    Copyright (c) 2022 Project CHIP Authors
+#    All rights reserved.
+#
+#    Licensed under the Apache License, Version 2.0 (the "License");
+#    you may not use this file except in compliance with the License.
+#    You may obtain a copy of the License at
+#
+#        http://www.apache.org/licenses/LICENSE-2.0
+#
+#    Unless required by applicable law or agreed to in writing, software
+#    distributed under the License is distributed on an "AS IS" BASIS,
+#    WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+#    See the License for the specific language governing permissions and
+#    limitations under the License.
+#
+
+import logging
+import time
+
+import chip.clusters as Clusters
+from chip import ChipDeviceCtrl
+from chip.ChipDeviceCtrl import CommissioningParameters
+from chip.interaction_model import Status
+from matter_testing_support import MatterBaseTest, async_test_body, default_matter_test_main
+from mobly import asserts
+
+
+class TC_ACE_1_5(MatterBaseTest):
+
+    def OpenCommissioningWindow(self) -> CommissioningParameters:
+        try:
+            params = self.th1.OpenCommissioningWindow(
+                nodeid=self.dut_node_id, timeout=600, iteration=10000, discriminator=self.matter_test_config.discriminators[0], option=1)
+            time.sleep(5)
```

Comment:
i think this was probably from another script. This was a workaround for a bug in the avahi dns-sd resolver that's now been fixed, so you don't need this. 

Given that a lot of the core tests are going to be using this type of function, it's probably worthwhile to put the whole function into the base, maybe with the controller as an arg.

Reply:
I did it, but I am not sure if you refer to `time.sleep(5)` about avahi dns-sd problem. The whole function I moved to into the base class without time.sleep but I am using the same parameters: timeout, iteration and discriminator.

Reply:
That's fine.